### ISSUE TITLE
[BUG] fixing quotes destroys content of attribute

### ISSUE THREAD

**Issue by georgmu** [yeonjuan/html-eslint#95]
Given the following html file:
```
<my-component list='["foo", "bar"]'></my-component>
```
with recommended options (including `@html-eslint/quotes`), I get the following warning: 

```
foo.html
  1:16  error  Expected double(") quotes but found single(')  @html-eslint/quotes
```

**Actual behavior:**

When running with '--fix', the following happens with the file:
```
<my-component list="]"""></my-component>
```

**Expected behavior:**
Not sure what exactly to do, there are several options:

If the intended external quote is found within the attribute value:
* do not warn
* warn, but do not fix
* swap quote style (`"`->`'` and `'`->`"`)
  * this would result in `list="['foo', 'bar']"`

**Comment by yeonjuan** [comment]
@georgmu  Thanks for reporting this issue. I can reproduce it - [demo](https://yeonjuan.github.io/html-eslint/playground#eyJjb2RlIjoiPG15LWNvbXBvbmVudCBsaXN0PSdbXCJmb29cIiwgXCJiYXJcIl0nPjwvbXktY29tcG9uZW50PiIsInJ1bGVzIjp7IkBodG1sLWVzbGludC9pbmRlbnQiOiJlcnJvciIsIkBodG1sLWVzbGludC9yZXF1aXJlLWxpLWNvbnRhaW5lciI6Im9mZiIsIkBodG1sLWVzbGludC9uby1leHRyYS1zcGFjaW5nLWF0dHJzIjoib2ZmIiwiQGh0bWwtZXNsaW50L3F1b3RlcyI6Im9mZiJ9fQ==). I'll fix it as soon

**Comment by georgmu** [comment]
Nice! Thank you very much!
